apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ProjectMetricsResourceTest.java

```diff
@@ -363,9 +352,34 @@ public static Stream<Arguments> invalidParameters() {
                             .intervalEnd(now)
                             .build()), ProjectMetricsService.ERR_START_BEFORE_END),
                     arguments(named("not supported metric", validReq.toBuilder()
-                            .metricType(MetricType.TOKEN_USAGE)
+                            .metricType(MetricType.DURATION)
                             .build()), ProjectMetricsService.ERR_PROJECT_METRIC_NOT_SUPPORTED.formatted(
-                                    MetricType.TOKEN_USAGE)));
+                                    MetricType.DURATION)));
+        }
+
+        @ParameterizedTest
+        @EnumSource(TimeInterval.class)
+        void emptyData(TimeInterval interval) {
+            // setup
+            mockTargetWorkspace();
+
+            Instant marker = getIntervalStart(interval);
+            String projectName = RandomStringUtils.randomAlphabetic(10);
+            var projectId = projectResourceClient.createProject(projectName, API_KEY, WORKSPACE_NAME);
+
+            Map<String, Integer> emptyTraces = new HashMap<>() {
+                {
+                    put(ProjectMetricsDAO.NAME_TRACES, null);
+                }
+            };
+
+            getMetricsAndAssert(projectId, ProjectMetricRequest.builder()
+                    .metricType(MetricType.TRACE_COUNT)
+                    .interval(interval)
+                    .intervalStart(subtract(marker, 4, interval))
```

Comment:
Does `4` mean that you take 4 intervals? If yes, then why 4?)
Guess would be better to replace magic number with a names constant. Same in other places

Reply:
Done in https://github.com/comet-ml/opik/pull/728/commits/d86602a52b2d26e3b3de97973bf910b2bccf24d1